src/django/python/settings.py

```diff
@@ -110,6 +110,6 @@
 
 
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/2.1/howto/static-files/
+# https://docs.djangoproject.com/en/4.1/howto/static-files/
```

Comment:
Might be better if this was django 3.X as that's what the app produces currently